### ISSUE TITLE
GitHub stats not provided anymore for users / organizations 

### ISSUE THREAD

**Issue by kcgthb** [squidfunk/mkdocs-material#3069]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

It looks like the feature introduced in #342 to allow getting user/org stats from GitHub (when the `repo_url` in `mkdosc.yml` is a user/organization URL rather than a repo URL) is not working anymore.

With this in `mkdocs.yml`:
```
repo_url:  https://github.com/orgname
```
the requested API URL (captured in the browser's Developer Tools) is https://api.github.com/repos/orgname instead of https://api.github.com/users/orgname.

### Expected behaviour

GitHub user/org stats are fetched from https://api.github.com/users/name

### Actual behaviour

GitHub user/org stats are fetched from https://api.github.com/repos/name

### Steps to reproduce

1. Set a user or organization URL as `repo_url` in `mkdocs.yml`
2. render the pages
3. no GitHub stats are provided

### Package versions

- Python: 3.9.7
- MkDocs: 1.2.2
- Material: 7.3.0

### Configuration

```yaml
repo_url:  https://github.com/orgname
```

### System information

- Operating system: Linux
- Browser: tested with up to date versions of Firefox, Chrome, Brave

**Comment by squidfunk** [comment]
Thanks for reporting. Fixed in ef0979f95.

It's a regression since 7.1.0, as it broke 6 months ago when the [merge](https://github.com/squidfunk/mkdocs-material/commit/ca3da9e3ca42c08f3ec9de8f93e1bb6f87daf93f) related to the 'Black Pearl' funding goal was done.

**Comment by kcgthb** [comment]
Fantastic, thank you for the quick fix!

**Comment by squidfunk** [comment]
Released as part of 7.3.1.
